plugins/search-react/src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -169,17 +189,22 @@ export const SelectFilter = (props: SearchFilterComponentProps) => {
     } = e;
 
     setFilters(prevFilters => {
-      const { [name]: filter, ...others } = prevFilters;
-      return value ? { ...others, [name]: value as string } : others;
+      const { [props.name]: filter, ...others } = prevFilters;
+
+      return (value as any[]).length === 0
+        ? others
+        : { ...others, [props.name]: value as T };
     });
   };
 
+  const displayEmpty = !label;
```

Comment:
Rather, wouldn't it be `displayEmpty = !placeholder`? When we have label it renders great, but when label and placeholder are defined they overlap...

Reply:
they should not as placeholder will not be rendered
```ts
if (displayEmpty && (selected as any[]).length === 0) {
            return placeholder;
}
```
I thought that way: label is more prioritised thing, so if label defined then to show it rather than placeholder